server/db/MockDB.go

```diff
@@ -0,0 +1,114 @@
+package db
+
+import (
+	"errors"
+
+	"github.com/google/uuid"
+	"github.com/jak103/uno/model"
+)
+
+// MockDB is an implemenation declaring the unit test db
+type mockDB struct {
+	games         map[string]model.Game
+	gamePasswords map[string]model.Game
+	players       map[string]model.Player
+}
+
+//type mongoDB struct {
+//	client   *mongo.Client
+//	uri      string
+//	database *mongo.Database
+//	games    *mongo.Collection
+//	players  *mongo.Collection
+//}
```

Comment:
Can this commented out code be removed? If not, why not?

Reply:
hmm... didn't notice that left in there. removing now.